src/main/java/potaton/Potaton.java

```diff
@@ -0,0 +1,299 @@
+                + "| (    )|| (   ) |   ) (   | (   ) |   ) (   | (   ) ||  \\  ( |\\\n"
+                + "| (____)|| |   | |   | |   | (___) |   | |   | |   | ||   \\ | |\n"
+                + "|  _____)| |   | |   | |   |  ___  |   | |   | |   | || (\\ \\) | \n"
+                + "| (      | |   | |   | |   | (   ) |   | |   | |   | || | \\   | \n"
+                + "| )      | (___) |   | |   | )   ( |   | |   | (___) || )  \\  | \n"
+                + "|/       (_______)   )_(   |/     \\|   )_(   (_______)|/    )_) \n";
+
+        System.out.println("_____________________________________________\n"
+                + logo + "\n"
+                + "Hello! I'm Potaton\n"
+                + "What can I do for you?\n"
+                + "_____________________________________________\n"
+        );
+    }
+
+    private static void addEvent(String line, ArrayList<Task> tasks) {
+        String arg = line.split(" ", 2)[1];
+        String event = arg.split("/at ", 2)[0];
+        String eventDate = arg.split("/at ", 2)[1];
+        Task t = new Events(event, eventDate);
+        tasks.add(t);
+        System.out.println("_____________________________________________\n"
+                + "Got it. I've added this task: \n"
+                +  t + "\n"
+                + "Now you have " + tasks.size() + " tasks in the list." + "\n"
+                + "_____________________________________________\n"
+        );
+    }
+
+    private static void loadEvent(String line, ArrayList<Task> tasks) {
+        String arg = line.split("] ", 2)[1];
+        String event = arg.split(" \\(at: ", 2)[0];
+        String eventDate = arg.split(" \\(at: ", 2)[1].split("\\)")[0];
+        Task t = new Events(event, eventDate);
+        tasks.add(t);
+        System.out.println("_____________________________________________\n"
+                + "Got it. I've added this task: \n"
+                +  t + "\n"
+                + "Now you have " + tasks.size() + " tasks in the list." + "\n"
+                + "_____________________________________________\n"
+        );
+    }
+
+    private static void addDeadline(String line, ArrayList<Task> tasks) {
+        String arg = line.split(" ", 2)[1];
+        String task = arg.split("/by ", 2)[0];
+        String dueDate = arg.split("/by ", 2)[1];
+        Task t = new Deadline(task, dueDate);
+        tasks.add(t);
+        System.out.println("_____________________________________________\n"
+                + "Got it. I've added this task: \n"
+                +  t + "\n"
+                + "Now you have " + tasks.size() + " tasks in the list." + "\n"
+                + "_____________________________________________\n"
+        );
+    }
+
+    private static void loadDeadline(String line, ArrayList<Task> tasks) {
+        String arg = line.split("] ", 2)[1];
+        String task = arg.split(" \\(by: ", 2)[0];
+        String dueDate = arg.split(" \\(by: ", 2)[1].split("\\)")[0];
```

Comment:
`2` appears to be a [magic number](https://nus-cs2113-ay2122s1.github.io/website/se-book-adapted/chapters/codeQuality.html#avoid-magic-numbers) which you could avoid. How about trying named constants?